<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>698</width>
    <height>541</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="centralGrid" native="true">
    <property name="geometry">
     <rect>
      <x>-10</x>
      <y>10</y>
      <width>801</width>
      <height>541</height>
     </rect>
    </property>
    <widget class="DrawArea" name="drawArea" native="true">
     <property name="geometry">
      <rect>
       <x>430</x>
       <y>60</y>
       <width>202</width>
       <height>202</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>202</width>
       <height>202</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>202</width>
       <height>202</height>
      </size>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QWidget" name="verticalLayoutWidget_2">
     <property name="geometry">
      <rect>
       <x>410</x>
       <y>340</y>
       <width>244</width>
       <height>91</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="layoutRight">
      <item>
       <layout class="QHBoxLayout" name="layoutHeaterOnOff">
        <item>
         <widget class="QLabel" name="labelHeaterStatus">
          <property name="font">
           <font>
            <family>Gadugi</family>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Heater:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="HeaterOn">
          <property name="font">
           <font>
            <family>Gadugi</family>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="text">
           <string>On</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="HeaterOff">
          <property name="font">
           <font>
            <family>Gadugi</family>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Off</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPushButton" name="stopSimulation">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <family>Gadugi</family>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Stop</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="Line" name="line">
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>10</y>
       <width>20</width>
       <height>451</height>
      </rect>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <property name="lineWidth">
      <number>3</number>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>318</width>
       <height>411</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="layoutLeft">
      <item>
       <widget class="QLabel" name="labelPrepPanel">
        <property name="font">
         <font>
          <family>Leelawadee UI Semilight</family>
          <pointsize>14</pointsize>
          <bold>false</bold>
         </font>
        </property>
        <property name="text">
         <string>Preparation panel</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QGridLayout" name="layoutSimParams">
        <property name="verticalSpacing">
         <number>10</number>
        </property>
        <item row="4" column="0" alignment="Qt::AlignHCenter">
         <widget class="QLabel" name="labelThreadsNumber">
          <property name="font">
           <font>
            <family>Gadugi</family>
            <pointsize>11</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Number of threads to use (up to 10)</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0" alignment="Qt::AlignHCenter">
         <widget class="QLabel" name="labelPowerSupply">
          <property name="font">
           <font>
            <family>Gadugi</family>
            <pointsize>11</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Power supply (in kWh) : </string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="5" column="0" alignment="Qt::AlignHCenter">
         <widget class="QSpinBox" name="threadsNumber">
          <property name="font">
           <font>
            <family>Gadugi</family>
            <pointsize>11</pointsize>
           </font>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <property name="buttonSymbols">
           <enum>QAbstractSpinBox::UpDownArrows</enum>
          </property>
          <property name="maximum">
           <number>10</number>
          </property>
          <property name="value">
           <number>1</number>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QDial" name="powerDial">
          <property name="maximum">
           <number>100</number>
          </property>
          <property name="value">
           <number>30</number>
          </property>
          <property name="sliderPosition">
           <number>30</number>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QSlider" name="PenSize">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <family>Gadugi</family>
           </font>
          </property>
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="labelSetPenSize">
          <property name="font">
           <font>
            <family>Gadugi</family>
            <pointsize>11</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Draw pen size: 1</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignBottom|Qt::AlignHCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="layoutStoveMaterial">
        <property name="spacing">
         <number>0</number>
        </property>
        <item alignment="Qt::AlignHCenter|Qt::AlignTop">
         <widget class="QLabel" name="labelTopMaterial">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <family>Gadugi</family>
            <pointsize>11</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Stove top material: </string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QGridLayout" name="layoutMaterials">
          <item row="1" column="1">
           <widget class="QPushButton" name="topBrick">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="font">
             <font>
              <family>Gadugi</family>
              <pointsize>11</pointsize>
             </font>
            </property>
            <property name="autoFillBackground">
             <bool>false</bool>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color: rgb(170, 255, 127);</string>
            </property>
            <property name="text">
             <string>Brick</string>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QPushButton" name="topIron">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="font">
             <font>
              <family>Gadugi</family>
              <pointsize>11</pointsize>
             </font>
            </property>
            <property name="autoFillBackground">
             <bool>false</bool>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color: rgb(255, 170, 127);</string>
            </property>
            <property name="text">
             <string>Iron</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QPushButton" name="topCopper">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="font">
             <font>
              <family>Gadugi</family>
              <pointsize>11</pointsize>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color: rgb(255, 170, 0);</string>
            </property>
            <property name="text">
             <string>Copper</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QPushButton" name="topSilver">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="font">
             <font>
              <family>Gadugi</family>
              <pointsize>11</pointsize>
             </font>
            </property>
            <property name="autoFillBackground">
             <bool>false</bool>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color: rgb(255, 85, 0);</string>
            </property>
            <property name="text">
             <string>Silver</string>
            </property>
           </widget>
          </item>
          <item row="1" column="2">
           <widget class="QPushButton" name="topGlass">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="font">
             <font>
              <family>Gadugi</family>
              <pointsize>11</pointsize>
             </font>
            </property>
            <property name="autoFillBackground">
             <bool>false</bool>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color: rgb(85, 255, 0);</string>
            </property>
            <property name="text">
             <string>Glass</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QPushButton" name="topQuartz">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="font">
             <font>
              <family>Gadugi</family>
              <pointsize>11</pointsize>
             </font>
            </property>
            <property name="autoFillBackground">
             <bool>false</bool>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color: rgb(255, 246, 123);</string>
            </property>
            <property name="text">
             <string>Quartz</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="layoutSlowFast">
        <item>
         <widget class="QLabel" name="labelSlow">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <family>Gadugi</family>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">color: rgb(255, 0, 0);</string>
          </property>
          <property name="text">
           <string>SLOW -</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="labelFast">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <family>Gadugi</family>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">color: rgb(0, 170, 0);</string>
          </property>
          <property name="text">
           <string>&gt; FAST calculations</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPushButton" name="startSimulation">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <family>Gadugi</family>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Start</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>450</x>
       <y>20</y>
       <width>171</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Gadugi</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Draw something</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>698</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuMain">
    <property name="title">
     <string>Main</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionExit"/>
   </widget>
   <addaction name="menuMain"/>
  </widget>
  <widget class="QStatusBar" name="statusbar">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
  </widget>
  <action name="actionNew">
   <property name="text">
    <string>New</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>DrawArea</class>
   <extends>QWidget</extends>
   <header>mainwindow.h</header>
   <container>1</container>
   <slots>
    <signal>mouseMoved(QPoint)</signal>
   </slots>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
